spec/requests/assignments_controller_spec.rb

```diff
@@ -19,6 +19,52 @@
     end
   end
 
+  describe "#show" do
```

Comment:
This should be in a different file now as it is to do with the internal contacts controller?